tests/test_pubsub.py

```diff
@@ -474,7 +474,7 @@ class TestPubSubSubcommands:
     def test_pubsub_channels(self, r):
         p = r.pubsub()
         p.subscribe('foo', 'bar', 'baz', 'quux')
-        for i in range(4):
+        for _ in range(4):
```

Comment:
Good catch! But personally speaking, I don't prefer irrelevant changes being included in this PR. What do you think @andymccurdy

Reply:
I am very new contributing in open source project. So please forgive my silly question. Are you saying here that resolving this Pylint warning should not be done in this PR because it is not its main focus? I would have thought we should use every opportunity to reduce the technical debt, especially if it is clear that it will not break anything.